site/src/Librecores/ProjectRepoBundle/Resources/views/Project/settings.html.twig

```diff
@@ -44,6 +44,12 @@
       <h3 class="panel-title">Description</h3>
     </div>
     <div class="panel-body">
+      <p><i>This is the description that will be visible on the project description page. You have the choice between
+      loading it from the README file in your source repository, or editing it manually.</i></p>
```

Comment:
Just README? Maybe makes sense to explicitly mention the support status for common file names like `README.md`, `README.asc`, `README.txt`, etc